README.md

```diff
@@ -26,18 +26,24 @@ gpu: 0                                 # gpu id to process on
 dataset: 
   num_chunks: 100000                   # newest nof chunks to parse
   train_ratio: 0.90                    # trainingset ratio
-  input: '/path/to/chunks/*/draw/'     # supports glob
+  # For separated test and train data.
+  input_train: '/path/to/chunks/*/draw/' # supports glob
+  input_test: '/path/to/chunks/*/draw/'  # supports glob
+  # For a one-shot run with all data in one directory.
+  # input: '/path/to/chunks/*/draw/'
 
 training:
     batch_size: 2048                   # training batch
     total_steps: 140000                # terminate after these steps
-    test_steps: 2000                   # number of steps between test evaluations.
+    test_steps: 2000                   # eval test set values after this many steps
+    total_steps: 140000                # terminate after these steps
+    # checkpoint_steps: 10000          # optional frequency for checkpointing before finish
     shuffle_size: 524288               # size of the shuffle buffer
     lr_values:                         # list of learning rates
         - 0.02
         - 0.002
         - 0.0005
-    lr_boundaries:                     # list of boundaries
+    lr_boundaries:                     # list of boundaries (boundaries are absolute to number of trained steps)
```

Comment:
'Technically' they are absolute modulo total_steps - not sure if we want that detail here or not.  total_steps is also similar, it runs until step count is a multiple of total_steps.
(I don't mind this text as is, just wanted to bring it up in case you have a better idea of what to say as a result.)

Reply:
Maybe an explanation somewhere of what the boundaries are in reference to. I want to clarify that they aren't relative to the current run of "total_steps" but absolute to how many steps have been run for the entire process.

Reply:
something like this under the config example:
* The LR boundaries are in reference to the overall number of steps trained (not relative to the current training run)

Reply:
The part that is potentially confusing is when the current step count is > total_steps.  Which doesn't happen so much in a bootstrap scenario perhaps, but is normal for our ongoing training process.

It can wait till a follow up PR though.